runtime/builtin_types_test.go

```diff
@@ -433,3 +440,12 @@ func newTestIndexObject(index int) *Object {
 	}))
 	return newObject(indexType)
 }
+
+func newTestAddObject(x int) *Object {
+	addType := newTestClass("Add", []*Type{ObjectType}, newStringDict(map[string]*Object{
+		"__add__": newBuiltinFunction("__add__", func(f *Frame, _ Args, _ KWArgs) (*Object, *BaseException) {
+			return NewInt(x + 1).ToObject(), nil
+		}).ToObject(),
+	}))
+	return newObject(addType)
```

Comment:
Rather than create a new helper, you can just create addType at the top of the test and then pass newObject(addType) to each of the relevant test cases. I try to avoid helpers unless they eliminate a significant amount of duplication.